content/doc_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package content
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"istio.io/istio.io/pkg/test/istioio"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/resource/environment"
+)
+
+var (
+	inst istio.Instance
+
+	testsToRun   = os.Getenv("TEST")
+	testEnv      = os.Getenv("ENV")
+	runAllTests  = (testsToRun == "")
+	testsAsSlice = split(testsToRun)
+
+	setupTemplate = `
+		source "${REPO_ROOT}/content/%v" # snips.sh
+		source "${REPO_ROOT}/tests/util/samples.sh"
+		source "${REPO_ROOT}/tests/util/verify.sh"
+		cd ${REPO_ROOT}
+	`
+
+	snipsFileSuffix = "/snips.sh"
+	testFileSuffix  = "/test.sh"
+	testCleanupSep  = "#! cleanup"
```

Comment:
```suggestion
	testCleanupSep  = "# !cleanup"
```
Don't use `#!` since it's a very special Unix character sequence called `shebang`. From Wikipedia:
```
The shebang is actually a human-readable instance of a magic number in the executable file, the magic byte
string being 0x23 0x21, the two-character encoding in ASCII of #!. This magic number is detected by the "exec"
family of functions, which determine whether a file is a script or an executable binary. The presence of the
shebang will result in the execution of the specified executable, usually an interpreter for the script's language. 
```

Reply:
How about `### cleanup ###`? This might be even less confusing

Reply:
I think that looks like it could just be an ordinary comment, which it's not. I like David's suggestion `# @cleanup`. That makes it clear it's something special (for the framework), but not to be confused with something special for the bash interpreter.

Reply:
Sounds good! I'll go with `# @cleanup` then